.travis.yml

```diff
@@ -3,7 +3,6 @@ language: php
 sudo: false
 
 php:
- - 5.5
```

Comment:
So the most consistent thing to do would be
- marking PHP 5.5 as untested and "most likely working but not supported"
- updating phpunit and starting to do tests using PHP 7
- doing builds on PHP 5.6
?